src/test/java/kitchenpos/products/tobe/domain/test/ProductPriceTest.java

```diff
@@ -0,0 +1,53 @@
+package kitchenpos.products.tobe.domain.test;
+
+import kitchenpos.products.tobe.domain.ProductPrice;
+import kitchenpos.products.tobe.domain.exception.NullAndNegativePriceException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.math.BigDecimal;
+import java.util.stream.Stream;
+
+import static kitchenpos.products.tobe.domain.fixture.PriceFixture.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class ProductPriceTest {
+    private static Stream<Arguments> provideBigDecimalForIsNotNullAndMinusValue() {
+        return Stream.of(
+            Arguments.of((BigDecimal) null),
+            Arguments.of(MINUS_PRICE)
+        );
+    }
+
+    @Test
+    @DisplayName("상품의 가격을 등록할 수 있다.")
+    void test1() {
+        assertDoesNotThrow(
+            () -> new ProductPrice(PRICE)
+        );
+    }
+
+    @ParameterizedTest
+    @DisplayName("상품의 가격이 비어있거나 음수이면 NegativePriceException 예외 발생")
+    @MethodSource("provideBigDecimalForIsNotNullAndMinusValue")
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    @ValueSource(strings = "-1000")
    @NullSource
```